src/Workspace/ErrorWorkspacePlaceholder.jsx

```diff
@@ -1,21 +1,33 @@
 import React from 'react';
+import { useHistory } from 'react-router-dom/cjs/react-router-dom';
+import isEnabled from '../helpers/featureFlags';
 import './Workspace.less';
 import NotFoundSVG from '../img/not-found.svg';
 import { components } from '../params';
+import workspaceRegistrationConfig from '../localconf';
 
-class ErrorWorkspacePlaceholder extends React.Component {
-  render() {
-    const supportEmail = components.login?.email || 'support@datacommons.io';
-    return (
-      <div className='error-workspace-placeholder__error-msg'>
-        <h1>Error opening workspace...</h1>
-        <p>
-        Workspace access requires authorization. Please contact <a href={`mailto:${supportEmail}`}>{supportEmail}</a> for more information.
-        </p>
-        <NotFoundSVG />
-      </div>
-    );
+const ErrorWorkspacePlaceholder = () => {
+  const history = useHistory();
+  if (isEnabled('workspaceRegistration')) {
+    if (workspaceRegistrationConfig) {
+      // eslint-disable-next-line react/prop-types
+      history.push('/workspace/register');
+    } else {
+      // eslint-disable-next-line no-console
+      console.log('Unable to display registration form. Missing workspaceRegistrationConfig');
+    }
```

Comment:
I remember in the design doc, there is a page with button and some explanatory text to tell user why they cannot open a workspace and they need to request access for it. But you are re-directing user to the registration form directly in here. Has the design changed? 
Personally I think the current flow is a bit confusing to the user who got redirected directly to a form upon clicking workspace button

Reply:
Oh! That's a good catch. I think I understand what you mean, when you say it could be confusing for a new user when they suddenly look at this form. I agree with you. The idea in the document was either to welcome them with a paragraph and put a button, or do it along with a form to fill. So, the current design is still in line with the original design choice, but adding more context is what was missing.

Reply:
What I'm saying is I think implementation is not following details from its design doc. So update either of them or both to keep them in sync